lms/djangoapps/courseware/views/views.py

```diff
@@ -249,6 +248,28 @@ def course_info(request, course_id):
 
     Assumes the course_id is in a valid format.
     """
+    def get_last_accessed_courseware(course, request, user):
```

Comment:
@andy-armstrong The only reason I moved this was to make it more clear it could go away with course info.